optimize_versioned_docs/version-3.13.0/apis-tools/optimize-api/tutorial.md

```diff
@@ -0,0 +1,167 @@
+---
+id: optimize-api-tutorial
+title: Tutorial
+description: "Step through an example to view your existing clients, create a client, view a particular client's details, and delete a client."
+---
+
+In this tutorial, we'll step through examples to highlight the capabilities of the Optimize API, such as listing your existing dashboard IDs, or deleting a dashboard.
+
+## Getting started
+
+- You need authentication to access the API endpoints. Find more information [here](./optimize-api-authentication.md).
+- To properly execute the commands to list existing dashboard IDs and delete a dashboard, ensure you have [created a collection](../../components/userguide/collections-dashboards-reports.md) containing a [dashboard](../../components/userguide/creating-dashboards.md).
+
+## Prerequisites
+
+- If you haven't done so already, [create a cluster]($docs$/guides/assets/react-components/create-cluster).
+- Upon cluster creation, [create your first client]($docs$/guides/setup-client-connection-credentials). Ensure you check the Optimize client scope box.
+
+:::note
+Make sure you keep the generated client credentials in a safe place. The **Client secret** will not be shown again. For your convenience, you can also download the client information to your computer.
+:::
+
+- In this tutorial, we utilize a JavaScript-written [GitHub repository](https://github.com/camunda/camunda-api-tutorials) to write and run requests. Clone this repo before getting started.
+- Ensure you have [Node.js](https://nodejs.org/en/download) installed as this will be used for methods that can be called by the CLI (outlined later in this guide). Run `npm install` to ensure you have updated dependencies. Run `npm install` to ensure you have updated dependencies.
```

Comment:
```suggestion
- Ensure you have [Node.js](https://nodejs.org/en/download) installed as this will be used for methods that can be called by the CLI (outlined later in this guide). Run `npm install` to ensure you have installed dependencies. Run `npm update` to ensure you have updated dependencies.
```
Double sentence. Double sentence. But I think this is what you were after?

Reply:
Adjusted in latest commit 👍 Adjusted in latest commit 👍